gohttplib.go

```diff
@@ -57,6 +59,12 @@ func HandleFunc(cpattern *C.char, cfn *C.FuncPtr) {
 		wPtr := cpointers.Ref(unsafe.Pointer(&w))
 		// Call our C function pointer using our C shim.
 		C.Call_HandleFunc(C.ResponseWriterPtr(wPtr), &creq, cfn)
+		// release the C memory
```

Comment:
I am not sure if we should also free `creq`
If that is needed, I don't know how to do that :(